src/rapids_pre_commit_hooks/alpha_spec.py

```diff
@@ -72,11 +36,12 @@ def node_has_type(node, tag_type):
     return node.tag == f"tag:yaml.org,2002:{tag_type}"
 
 
-def is_rapids_cuda_suffixed_package(name):
-    return any(
-        (match := CUDA_SUFFIX_REGEX.search(name)) and match.group("package") == package
-        for package in RAPIDS_CUDA_SUFFIXED_PACKAGES
-    )
+def strip_cuda_suffix(name):
```

Comment:
```suggestion
def strip_cuda_suffix(name: str) -> name:
```

While you're touching this, could you please add type hints? To help type-checkers find bugs and humans to understand the data flow.